probe_test.go

```diff
@@ -626,3 +626,49 @@ func TestLicenseStatus(t *testing.T) {
 		t.Fatalf("metric compare: err %v", err)
 	}
 }
+
+func TestLinkStatus(t *testing.T) {
+	c := newFakeClient()
+	c.prepare("api/v2/monitor/system/link-monitor", "testdata/link-monitor.jsonnet")
+	r := prometheus.NewPedanticRegistry()
+	if !testProbe(probeLinkMonitor, c, r) {
+		t.Errorf("probeLinkMonitor() returned non-success")
+	}
+
+	em := `
+# HELP fortigate_link_bandwidth_rx Bandwidth available on this link for sending
```

Comment:
Add the unit as a suffix, like _bps for bit/s. I would also add "available".
So something like `fortigate_link_available_rx_bps` might be better? I am fine with `fortigate_link_bandwidth_rx_bps` as well if you prefer it the way it is.

Reply:
as the metric is called bandwidth in the backend, I prefer the second one.     
Like all other metrics in this API endpoint there is no documentation what these values mean.